### ISSUE TITLE
Accessibility feedback, design intent collaboration for COVID screening tool

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9497]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

This epic will contain the accessibility staging review findings issue tickets.

## Point of Contact

**VFS Point of Contact:** Jennifer, Trevor

cc @1Copenut ready for your 👀  — Thank you!

@artsymartha68 Feedback complete, itemized below. 

## Definition of Done

1. Acknowledge feedback
1. Document decisions made
1. Accessibility specialist will review and close ticket

**Comment by jenstrickland** [comment]
Rather than a single issue ticket for the findings, I'll provide guidance in individual numbered comments. 

<hr/>

1. The area above the h1 typically has the breadcrumbs component. This could have a link to Home, at a minimum. Otherwise, consider customizing the heading to add some whitespace before it.

**Comment by jenstrickland** [comment]
2. Please remove the smooth scroll from the automatic focus management, as it may trigger those with motion sensitivity.

**Comment by jenstrickland** [comment]
3. Consider adding semantic and using Source Sans for the questions. The sans-serif font will be easier for those with some cognitive considerations.

    The Benefits and Memorials 2 team has a radio input that enhances the clickable area affordance with what looks more like a button. 

    Design mockup: 
    https://vsateams.invisionapp.com/share/6MVTG94WNH5#/screens/407910007 

    Staging link: 
    https://staging.va.gov/hearing-aid-batteries-and-accessories/introduction

**Current code**

```html
<main id="main" class="main">

<!-- Generated from manifest.json -->

  <div class="section">
    <div id="react-root">
      <div class="vads-l-grid-container">
        <h1>COVID-19 screening tool</h1>
        <div class="va-introtext">
          <p>Please answer the 5 questions listed below. Share your results with the staff member at the facility entrance.</p><p>We won't store or share your data.</p>
        </div><!-- end va-introtext -->
        <div>
          <div>
            <div name="multi-question-form-0-scroll-element"></div>
            <div class="feature">
              <h2>In the past 24 hours, have you had a fever?</h2>
              <button type="button" class="usa-button-big  usa-button" value="yes">Yes</button>
              <button type="button" class="usa-button-big  usa-button-secondary" value="no">No</button>
            </div><!-- end feature -->
          </div>

          <!-- other questions -->

          <div class="feature covid-screener-results covid-screener-results-incomplete">
            <div name="multi-question-form-5-scroll-element"></div>
            <div>
              <div>Please answer all the questions above.</div>
            </div>
          </div>
        </div>

      </div><!-- end vads-l-grid-container -->
    </div> <!-- end react-root -->
  </div>
</main>
```

**Code to consider**

```diff
<main id="main" class="main">

<!-- Generated from manifest.json -->

  <div class="section">
    <div id="react-root">
      <div class="vads-l-grid-container">
        <h1>COVID-19 screening tool</h1>
        <div class="va-introtext">
          <p>Please answer the 5 questions listed below. Share your results with the staff member at the facility entrance.</p><p>We won't store or share your data.</p>
        </div><!-- end va-introtext -->
+       <form>
          <div>
            <div name="multi-question-form-0-scroll-element"></div>
+           <fieldset class="feature">
+             <legend id="screener_fever">In the past 24 hours, have you had a fever?</legend>
+             <label for="">
+               <input id="covid_fever_yes" type="radio" name="covid_fever" /> 
+               Yes
+             </label>
+             <label for="">
+               <input id="covid_fever_no" type="radio" name="covid_fever" />
+               No
+             </label>
+           </fieldset><!-- end feature -->
          </div>

          <!-- other questions -->

          <div class="feature covid-screener-results covid-screener-results-incomplete">
            <div name="multi-question-form-5-scroll-element"></div>
            <fieldset role="alert">
              <p>Please answer all the questions above.</p>
            </fieldset>
          </div>
+       </form>

      </div><!-- end vads-l-grid-container -->
    </div> <!-- end react-root -->
  </div>
</main>
```

**Comment by jenstrickland** [comment]
4. Consider semantic markup changes. These screen shots include semantic adjustments, as well as moving the buttons side-by-side to clarify the options cognitively. I think this would work for the walking scenario, but trust your judgement since you observed people going into the VAMC.

![Screen Shot 2020-05-26 at 3 25 57 PM](https://user-images.githubusercontent.com/57469/82942347-d0b11180-9f65-11ea-9b8c-5b7738f3053b.png)

![Screen Shot 2020-05-26 at 3 27 25 PM](https://user-images.githubusercontent.com/57469/82942359-d4dd2f00-9f65-11ea-8c6e-b60e500cf6dd.png)


![Screen Shot 2020-05-26 at 3 29 58 PM](https://user-images.githubusercontent.com/57469/82942364-d9094c80-9f65-11ea-9404-39d7d98da5f3.png)

**Comment by 1Copenut** [comment]
The only feedback I'd add to VSA's assessment would be to test with as many devices and screen readers as possible to ensure we are setting focus on the text blocks above our Yes | No inputs so the text is read back. We can help test JAWS and NVDA if the team doesn't have access to.

It will probably include custom code that fires a `focus()` helper, if not already baked into the screener tool.

**Comment by 1Copenut** [comment]
VSP agrees with VSA feedback. Thanks all!

**Comment by artsymartha68** [comment]
a11y experts, when I'm using VoiceOver on iOS, each item in the bulleted list is read aloud as "black square" then swipe right to hear the corresponding text. I don't think that's right....?

**Comment by artsymartha68** [comment]
Also: confirmed that the focus doesn't automatically go to the next question once I hit Yes or No. We'd like that if possible, makes it much easier to get through.

**Comment by artsymartha68** [comment]
I like the screen-wide buttons to reduce thumb reach. So I'd prefer not to have Yes/No side by side if we can help it.

**Comment by batemapf** [comment]
Opened #9600 #9601 #9604 #9606 #9608 to address above comments.

**Comment by jenstrickland** [comment]
> a11y experts, when I'm using VoiceOver on iOS, each item in the bulleted list is read aloud as "black square" then swipe right to hear the corresponding text. I don't think that's right....?

Sadly, yes, that is correct. I wish that the VA design system didn't use the square bullets, but it does.

**Comment by jenstrickland** [comment]
> Also: confirmed that the focus doesn't automatically go to the next question once I hit Yes or No. We'd like that if possible, makes it much easier to get through.

I'd like to test after the semantic markup changes happen. Moving focus after the person selects Yes or No is okay, as long as it is easy to move back if they need to change an answer.

**Comment by batemapf** [comment]
@jenstrickland not sure if this is ready to be closed. Looks like a bunch of tickets were created to address the items. Do those tickets need to be closed to close this ticket?

**Comment by jenstrickland** [comment]
As this was a "design intent" review, we often provide the feedback in comments. For a staging review, we write issue tickets and attach them to an epic.

We could turn this into an epic, and attach the issue tickets to this, to follow the progress.

**Comment by joshkimux** [comment]
@artsymartha68 I see your name assigned here! From reviewing the epic, it looks like all the issues Jen caught were closed out. This may be set to close 😄
